validator/remote/src/main/java/tech/pegasys/teku/validator/remote/EventSourceHandler.java

```diff
@@ -34,13 +38,28 @@
 
   private final JsonProvider jsonProvider = new JsonProvider();
   private final ValidatorTimingChannel validatorTimingChannel;
+  private final Counter connectCounter;
+  private final Counter disconnectCounter;
+  private final Counter invalidEventCounter;
+  private final Counter timeoutCounter;
+  private final Counter errorCounter;
 
-  public EventSourceHandler(final ValidatorTimingChannel validatorTimingChannel) {
+  public EventSourceHandler(
+      final ValidatorTimingChannel validatorTimingChannel, final MetricsSystem metricsSystem) {
     this.validatorTimingChannel = validatorTimingChannel;
+    final LabelledMetric<Counter> eventSourceMetrics =
+        metricsSystem.createLabelledCounter(
+            TekuMetricCategory.VALIDATOR, "event_stream", "Event stream status counters", "reason");
```

Comment:
hmm, despite what I said in conversations earlier I think we might have got this slightly wrong.  Labelled counters should make sense when you sum up all the values ignoring the labels.  I think that makes sense for `disconnect`, `timeout` and `error` but not `invalidEvent` or `connect`.

So I think I'd make this metric `event_stream_disconnections_total` and then add a new `event_stream_invalid_events_total` metric and either drop the `connect` metric as it's not particularly useful or make it its own `event_stream_connects_total` metric.